pkgs/applications/misc/collision/default.nix

```diff
@@ -10,18 +10,39 @@
 , openssl
 , libxml2
 , pkg-config
+, gitUpdater
+, _experimental-update-script-combinators
+, runCommand
+, crystal2nix
+, writeShellScript
 }:
 crystal.buildCrystalPackage rec {
   pname = "Collision";
-  version = "3.6.0";
+  version = "3.8.0";
 
   src = fetchFromGitHub {
     owner = "GeopJr";
     repo = "Collision";
     rev = "v${version}";
-    hash = "sha256-Qxe4ILDdfYfxu95EvKRTvkAOgDIZDiLymBlZouBWn0M=";
+    hash = "sha256-Bo/u0UYM/N7tLqdCs2OU5pdj2s9LXPooSR1PCGk9dSc=";
   };
-  patches = [ ./make.patch ];
+
+  postPatch = ''
+    substituteInPlace Makefile \
+      --replace-fail './bin/gi-crystal || $(CRYSTAL_LOCATION)shards install && ./bin/gi-crystal' gi-crystal \
```

Comment:
Since we're building gi-crystal from a shard, we don't need this patch (and we don't need the gi-crystal package as build input).
```suggestion
```

Reply:
Done